### PR TITLE
Redesign homepage

### PR DESCRIPTION
**Pull request by waltersajtos**
## Description

Implementation of the new homepage designs

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [ ] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the homepage
2. Check if all buttons redirect to the expected page
3. Check if everything looks good on all resolutions
4. Check if the modals open on the homepage and work as expected

## Link to issue

Closes: #380

### CODE REVIEW

**Comment by niraymak** [comment]
Any changes made?

Noticed that one highlight was not showing properly in responsive view.

**Comment by waltersajtos** [comment]
> 
> 
> Any changes made?
> 
> Noticed that one highlight was not showing properly in responsive view.
> image

I only changed the highlight logic, no styling. @liekevdvoort was going to look at it iirc.

**Comment by niraymak** [comment]
Except for the highlights (highlight description and title were showing on the same place at some resolutions). 

I think they will move to a carrousel view, right?

**Comment by niraymak** [comment]
Not sure if this can and should be fixed, but when loading the page for a split second it looks like this: 
image

The background image is stretched and the text moves from the center to the left. This probably has something to do with the highlights loading in.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Added search bar inside navbar with autocomplete suggested results. - [#403](https://github.com/DigitalExcellence/dex-frontend/issues/403)
+
```
Comment (niraymak) [inline CHANGELOG.md:13]: Missing changelog for homepage.

**Review by niraymak — CHANGES_REQUESTED** [review]
Good work, excited for this one! 🥇 

Had one remark: when you create a highlight it is required to at least enter 75 characters.
I added one with around 75 characters, which was not completly readable for me. Maybe we should change the requirements?

image


Please also update changelog.

**Review by niraymak — DISMISSED** [review]

**Review by niraymak — DISMISSED** [review]

**Review by niraymak — APPROVED** [review]
🥇

**Review by RubenFricke — APPROVED** [review]
Great! Well done 🎉

**Pull request merged**